Favor/FavorNetworkKit/Sources/FavorNetworkKit/APIError.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  File.swift
+//  
```

Comment:
아 여기 자동으로 내용이 안붙더라고..
아마 어딘가에 설정 방법이 있긴 할텐데
일단은 직접 넣어주자...!

```suggestion
//  APIError.swift
//  Favor
```